### PR TITLE
SOC-5361 [social-rdbms] My connection page is too slow

### PR DESCRIPTION
**Pull request by boubaker**
Add new DTO for AllPeople page to retrieve Identity and Relationship at the same time.
Avoid to modify filter with viewer as excluded identity. (It's excluded anyway from queries)

### CODE REVIEW

File: component/core/src/main/java/org/exoplatform/social/core/identity/model/IdentityWithRelationship.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.exoplatform.social.core.identity.model;
+
+import org.exoplatform.social.core.profile.ProfileLoader;
+import org.exoplatform.social.core.relationship.model.Relationship;
+
+public class IdentityWithRelationship extends Identity {
```
Comment (thomasdelhomenie) [inline component/core/src/main/java/org/exoplatform/social/core/identity/model/IdentityWithRelationship.java]: Can you add a comment on the class to explain why we need it please ?

File: component/core/src/main/java/org/exoplatform/social/core/profile/ProfileFilter.java [hunk 2]
```diff
@@ -50,6 +50,9 @@
   /** the list of remoteId who online on system**/
   private List<String> onlineRemoteIds;
 
+  /** Current viewer identity */
+  private Identity currentIdentity;
```
Comment (thomasdelhomenie) [inline component/core/src/main/java/org/exoplatform/social/core/profile/ProfileFilter.java]: "currentIdentity" does not make a lot of sense in this object. We already have a excludedIdentityList, isn't it sufficient ?

Comment (boubaker) [inline component/core/src/main/java/org/exoplatform/social/core/profile/ProfileFilter.java]: nope, currentViewerIdentity if you want, but excludedIdentityList is different

File: component/core/src/main/java/org/exoplatform/social/core/identity/model/IdentityWithRelationship.java (hunk 1 above)
Comment (boubaker) [inline component/core/src/main/java/org/exoplatform/social/core/identity/model/IdentityWithRelationship.java]: ok, will do

**Comment by boubaker** [comment]
Merged

**Pull request closed without merging**